src/components/edit-on-github-link/index.tsx

```diff
@@ -0,0 +1,22 @@
+import InlineSvg from '@hashicorp/react-inline-svg'
+import svgGithub from '@hashicorp/flight-icons/svg/github-16.svg?include'
```

Comment:
We can use the `<IconGithub16 />` component here

```ts
import { IconGithub16 } from '@hashicorp/flight-icons/svg-react/github-16'
```

Reply:
@BRKalow Glad you brought this up and curious to get your take on this - we talked a while back during Learn dev work (with Will) about the merits of rendering a React-ified SVG component, vs setting the SVG as inline HTML.

Previous concern with the React-ified SVG was performance... which kinda made sense to me, since React would have to care about eg each `path` element rather than treat the `svg` as a fixed, just-render-it thing. But that's with a pretty limited knowledge of the inner workings of React.

Do you think that perf concern is worth using React-ified SVGs only when we actually need re-render-within-the-SVG functionality? (Say, so we can set the `title` from outside the SVG for a117 purposes). Or is the performance concern so minor that we might as well just always use React-ified SVGs, and we'll be fine even if there are dozens or hundreds rendering on a page?

Reply:
I'd be curious to see at what point we would start experiencing performance issues, I'd be surprised if we ever encountered any with the number of icons we use 💭  I tried to do some initial research when setting up this approach, and there was nothing glaring that I ran into where folks encountered noticeable performance issues rendering SVGs as React elements vs. inlining them.

One thing to consider with the inline/string approach is that those blobs can't be minified as well, so ultimately it likely results in larger JS bundles. I think if we're trying to squeeze out performance, something which uses SVG `use` ([ref](https://css-tricks.com/svg-use-with-external-reference-take-2/)) would be the way to go.

Reply:
Oh awesome, I didn't realize you'd already looked into this 💯 Since the performance concern seems pretty minor / non-noticeable, will swap out for the React component, definitely feels better in terms of consistency & ease of use 👍